lib/formatters/tapFormatter.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/**
+ * @type {import('stylelint').Formatter}
+ */
+const tapFormatter = (results) => {
+	let lines = [`TAP version 13\n1..${results.length}`];
+
+	results.forEach((result, index) => {
+		lines.push(
+			`${result.errored ? 'not ok' : 'ok'} ${index + 1} - ${result.ignored ? 'ignored ' : ''}${
+				result.source
+			}`,
+		);
+
+		if (result.warnings.length > 0) {
+			lines.push('---', 'messages:');
```

Comment:
I am not sure but it seems to me that `\n` is missing.

> `YAMLBlock   := "  ---\n" (YAMLLine)* "  ...\n"`

Reply:
See `return lines.join('\n');` at the end of the function.